Volume/Views/Reads/MagazinesView.swift

```diff
@@ -48,52 +58,63 @@ struct MagazinesView: View {
                 await viewModel.fetchContent()
             }
         }
+        .refreshable {
+            Task {
+                await viewModel.refreshContent()
+            }
+        }
     }
     
     private var searchBar: some View {
-        NavigationLink(destination: SearchView(), label: {
+        NavigationLink {
+            SearchView()
+        } label: {
             SearchBar(searchState: $viewModel.searchState, searchText: $viewModel.searchText)
                 .disabled(true)
                 .padding(.init(top: Constants.searchTop, leading: Constants.sidePadding, bottom: 0, trailing: Constants.sidePadding))
-        })
+        }
     }
     
     private var featuredMagazinesSection: some View {
-        Group {
-            Header("Featured")
-                .padding([.top, .horizontal])
-            
-            ScrollView(.horizontal, showsIndicators: false) {
-                LazyHStack(spacing: Constants.magazineHorizontalSpacing) {
-                    switch viewModel.featuredMagazines {
-                    case .none:
-                        ForEach(0..<3) { _ in
-                             MagazineCell.Skeleton()
-                        }
-                    case .some(let magazines):
-                        ForEach(magazines) { magazine in
-                            NavigationLink {
-                                MagazineReaderView(initType: .readyForDisplay(magazine), navigationSource: .featuredMagazines)
-                            } label: {
-                                MagazineCell(magazine: magazine)
+        Section {
+            if viewModel.featuredMagazines?.count == 0 {
+                VolumeMessage(image: Image.volume.magazine, message: .noFeaturedMagazines, largeFont: false, fullWidth: false)
```

Comment:
after looking at the demo video, im wondering if it would be better to just remove the featured section when there are no featured magazines instead of having the empty section/message at the top of the page. let me know your thoughts and make appropriate changes as you see fit

Reply:
Yea that's what Liam suggested as well and had that originally. I just thought it was weird having "More" magazines without a featured section, but I'll talk to Liam about it.

Reply:
ok no worries, we can leave it as is right now and come back to it if we want to make changes

Reply:
I went ahead and removed it as requested by Liam